code/__DEFINES/machines.dm

```diff
@@ -126,3 +126,8 @@
 #define AI_DISPLAY_MODE_BLANK 0
 #define AI_DISPLAY_MODE_EMOTE 1
 #define AI_DISPLAY_MODE_BSOD 2
+
+// Door operations
+#define OPENING 1
+#define CLOSING 2
+#define MALF 3
```

Comment:
Having these defines be common words in a define file is probably not a good idea, maybe something like this? It might be confusing with something like windoors that also use #defs named similar to this if I remember correctly
```suggestion
#define DOOR_OPENING 1
#define DOOR_CLOSING 2
#define DOOR_MALF 3
```